widgetastic_manageiq/__init__.py

```diff
@@ -1388,6 +1388,15 @@ def __locator__(self):
         )
 
 
+class SquashButton(Button):
+    """ This button requires a specific locator"""
+
+    def __locator__(self):
```

Comment:
I think this would be better implemented as `ROOT = ParametrizedLocator('.//div[contains(@class, "btn") and @id="squash_button" {@locator_conditions}]')